pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -7,18 +7,75 @@
 # include <string.h>
 # include <stdarg.h>
 # include <stdlib.h>
+# include <stdio.h>
+
+/*
+ * Takes an array, and prints it's elements the layed out in memory (similar to ravel)

```

Comment:
i think there is a typo here `the layed out`

Reply:
Yes!

Reply:
https://github.com/pyccel/pyccel/pull/1128/commits/f17b9e922db69508204674fcd9f61563e0167b02

Reply:
I only see 2 fixed in the PR. There is still the grammar error "the laid out" -> "laid out"

Reply:
Yeh
https://github.com/pyccel/pyccel/pull/1128/commits/9a9e82a1b966e31dd7b9ebe43fd7063480fd9994